src/Microsoft.DotNet.XHarness.Android/AndroidDevice.cs

```diff
@@ -3,19 +3,24 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.Text.Json.Serialization;
 
 namespace Microsoft.DotNet.XHarness.Android;
 
 public record AndroidDevice
 {
     public string DeviceSerial { get; init; }
 
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
Couldn't this be configured once through `DefaultIgnoreCondition` property on serializer?

Reply:
Yeah, I think that will work for all the other models as well